src/locales/fr.ts

```diff
@@ -16,7 +16,8 @@ export default {
 		h2: "Le Hackathon de la Capitale Canadienne",
 		h3: "Université d'Ottawa ",
 		h3_p2: " Université Carleton",
-		more: "En Savoir Plus",
+		more: "Suivez le piratage!",
```

Comment:
This should probably say something like "Suivez le hackathon" because piratage doesn't really mean the same thing in French.